src/components/layout/organize/SingleTaskLayout.tsx

```diff
@@ -0,0 +1,34 @@
+import { FunctionComponent } from 'react';
+import { useQuery } from 'react-query';
+import { useRouter } from 'next/router';
+
+import getTask from 'fetching/tasks/getTask';
+import TabbedLayout from './TabbedLayout';
+
+import TaskStatusText from 'components/organize/tasks/TaskStatusText';
+
+
+const SingleCampaignLayout: FunctionComponent = ({ children }) => {
+    const { taskId, orgId, campId } = useRouter().query;
+    const { data: task } = useQuery(['task', taskId ], getTask(orgId  as string, taskId as string));
+
+    if (!task) return null;
+
+    return (
+        <TabbedLayout
+            baseHref={ `/organize/${orgId}/campaigns/${campId}/tasks/${taskId}` }
+            defaultTab="/"
```

Comment:
There is something wrong here (probably inside `TabbedLayout`?) that is preventing the current tab from being highlighted as selected. See screenshot below (note URL, breadcrumbs and state of tab):

![image](https://user-images.githubusercontent.com/550212/126665419-63d19cab-8e4b-42d6-8ae9-10339f992f4c.png)